### ISSUE TITLE
Aten does not load force field files

### ISSUE THREAD

**Issue by sturgman** [trisyoungs/aten#14]
I am working with Windows 7 and version 2.1. I have reproduced this with the my own force field files and with force field files that come standard with aten. When I select a file to open, nothing happens in aten. No error message. No force field loaded.

**Comment by alinelena** [comment]
Hi, 

can you share the forcefield file in question? if not public can you email it?alin@elena.space

Alin

**Comment by sturgman** [comment]
Hey Alin,

You can use the opls force field that comes with aten. My custom one is:

```
# saft-mie force field from Herdes, Totton, and Muller (Fluid Phase Equilibria) 2015
# Coarse grained force field for the molecular simulation of naturalgases and condensates

name "saft mie"
units K

uatypes
1       T       C	46.07       "nbonds=1 ~C(nbonds=1)" "United atom toluene"
2	D_mid	C	42.585	    "nbonds=2 ~C(nbonds=2) ~C(nbonds=1)" "United atom mid dodecane"
3	D_end	C	42.585	    "nbonds=1 ~&2" "United atome end dodecane"
4	W	O	18.015	    "nbonds=0"	   "United atom water"
end

#TYP    AT      CHARGE  EPSILON SIGMA
inter lj
1       T       0.0     411.87  4.266
2	D_mid	0.0	378.56	4.351
3	D_end	0.0	378.56	4.351
4	W	0.0	378.87	2.915
end

bonds constraint
T     T		2845    4.266
D_mid D_mid	2845	4.351
D_mid D_end	2845	4.351
end

angles harmonic
D_mid D_mid D_end 0.8130463 157.6
end
```

**Comment by trisyoungs** [comment]
Utterly dumb mistake by me - forcefields code was trying to get filename(s) from the open grids dialog.  Fixed in 2f31b0f5e8bf651a5a7dbd627523ed6420c5a0d0.
